conjureup/controllers/newcloud/common.py

```diff
@@ -87,51 +87,57 @@ def save_creds(cloud, credentials):
 def is_lxd_ready():
     """ routine for making sure lxd is configured for localhost deployments
     """
-    if utils.lxd_version() < parse_version('2.9'):
-        return {"ready": False,
-                "msg": "The current version of LXD found on this system is "
-                "not compatible. Please run the following to get the latest "
-                "supported LXD:\n\n"
-                "  sudo apt-add-repository ppa:ubuntu-lxc/lxd-stable\n"
-                "  sudo apt-get update\n"
-                "  sudo apt-get install lxd lxd-client\n\n"
-                "Or if you're using the snap version:\n\n"
-                "  sudo snap refresh lxd\n\n"
-                "Once complete please re-run conjure-up."}
-
-    if not utils.check_user_in_group('lxd'):
-        return {"ready": False,
-                "msg": "User {} is not part of the LXD group. You will need "
-                "to exit conjure-up and do one of the following:\n\n"
-                " 1: Run `newgrp lxd` and re-launch conjure-up\n\n"
-                "  Or\n\n"
-                " 2: Log out completely, Log in and "
-                "re-launch conjure-up".format(os.environ['USER'])}
-
     try:
-        setup_lxdbr0_network()
-        setup_conjureup0_network()
+        lxd_init()
```

Comment:
This is doing a lot of stuff in a blocking fashion.  Should we render an interstitial while this is going on, or are we confident that it will never take very long?